_posts/2025-11-11-cockpit-351.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cockpit 351
+author: mvo
+date: '2025-11-11'
+tags: cockpit, machines
+slug: cockpit-351
+category: release
+summary: 'deleting firewall ports, restarting VMs, forwarding ports'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 351, and cockpit-machines 344:
+
+
+## Firewall: Additional ports can now be deleted
+
+"Additional ports" in a firewall zone are now listed each in their
+individual row, and can be individually deleted.
+
+Thanks a lot libkurisu for this contribution!
+
+<img width="720" height="234" alt="image" src="https://github.com/user-attachments/assets/a50977ba-f404-4784-96be-cdbc17eaf9c5" />
```

Comment:
Didn't we fix this so it grabs the new format too?

Reply:
Yes, I was thinking the same... I probably just haven't rebased...

Reply:
Its fixed in the Python version, not in ruby yet https://github.com/cockpit-project/cockpit-project.github.io/pull/837

Reply:
I put in a new screenshot manually, with a open dropdown to show the "Delete" action.